### ISSUE TITLE
Dr. Nick from the Simpson's

### ISSUE THREAD

**Issue by nicolasfoss** [rfordatascience/tidytuesday#773]
Hey Jon!

Has TT ever looked at Simpson's?

I thought of submitting a Simpsons's dataset that is focused on the character Dr. Nick (it would possibly give folks a chuckle because I'm sometimes called Dr. Nic by my friends).

Let me know what you think.

**Comment by jonthegeek** [comment]
Looks like we only did "Simpsons Guest Stars" in 2019:
``` r
# pak::pak("r4ds/ttmeta")
ttmeta::tt_summary_tbl |> 
  dplyr::filter(stringr::str_detect(tolower(title), "simpson"))
#> # A tibble: 1 × 8
#>    year  week date       title            source_title source_urls article_title
#>   <int> <int> <date>     <chr>            <chr>        <list>      <chr>        
#> 1  2019    35 2019-08-27 Simpsons Guest … Wikipedia    <chr [1]>   Wikipedia    
#> # ℹ 1 more variable: article_urls <list>
```

<sup>Created on 2024-11-24 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

We'd love a Dr. Nick dataset!

**Comment by nicolasfoss** [comment]
@jonthegeek say no more!

I will work on these datasets today.

I cloned the TT repo. Can I do a pull request from the cloned repo just as well?

**Comment by jonthegeek** [comment]
@nicolasfoss Absolutely! See [the PR instructions](https://github.com/rfordatascience/tidytuesday/blob/master/.github/pr_instructions.md) if you haven't already!

**Comment by nicolasfoss** [comment]
Hey @jonthegeek!

So, I didn't make it all about Dr. Nick because I noticed that the last Simpsons dataset was quite different from this one.  It was much smaller and all about guest characters.

For that reason, I didn't go with Dr. Nick and went with the full dataset as there are many insights to be gleaned.  This issue can be closed related to my pull request if you elect to accept the dataset.

Thanks for all you do and hope your weekend wraps up nicely.

**Comment by pinballelectronica** [comment]
I just came across this repo when looking for some niche datasets and I got giddy over what y'all are doing here.  I have transcribed the first 12 seasons of the Simpsons and pulled out various info from it including all Dr. Nick conversations if I can help. I've also done full transcriptions of (most) all of the Car Talk episodes including diarization and etc, built a chatbot to ask (both of them) car questions from beyond and etc.  Would love to help or join.

**Comment by nicolasfoss** [comment]
Hey @pinballelectronica 👋 

I have a pull request for the full Simpsons dataset from Kaggle. It has been around for a while now. 

What if you did the Dr. Nick idea I had?

I'm sure we won't get enough of the Simpsons!!

**Comment by pinballelectronica** [comment]
HI Everybody!  Sorry the holidays got the best, or worst, of me.   I'll do it.  

Note: will not officially accept Simpsons existed after Season 11.   🤣
